Cython/Utility/ModuleSetupCode.c

```diff
@@ -307,6 +307,12 @@
   #define __has_cpp_attribute(x) 0
 #endif
 
+/* This macro can be used to suppress warnings about constant conditional expressions */
+#ifndef CYTHON_CONDITION
+  int CYTHON_CONDITION_IMPL(int b) { return b; }
```

Comment:
This probably wants to be inline.

I also don't think it fits into the general naming scheme that Cython adopts (probably should start with `__Pyx`, all upper-case implies macro).

Reply:
I will update the name. Where should I place the macro, so it does has access to `CYTHON_INLINE`?

Reply:
Should the macro name start with `CYTHON` or `__PYX`?

Reply:
We use `CYTHON_*` for (user visible) configuration related macros and case variants of `__Pyx_*` for anything that's internal. This seems internal.